examples/generative/corrdiff/generate.py

```diff
@@ -91,6 +91,11 @@ def main(cfg: DictConfig) -> None:
     boundary_pix = getattr(cfg, "boundary_pixels", 2)
     hr_mean_conditioning = getattr(cfg, "hr_mean_conditioning", False)
 
+    # Initialize distributed manager
```

Comment:
This is fine, but I'd put it at the very top of main, just as a good safety practice.